packages/p2p/src/types/adverts.types.ts

```diff
@@ -0,0 +1,56 @@
+export type TAdvert = {
+    advert: TAdvertProps;
+};
+
+export type TAdvertProps = {
```

Comment:
If all of the properties are optional 😅  you could do a `Partial<TAdvertProps>`. But is this really what you want to do? Don't you want to enforce some types as required so that your code won't break?

Reply:
i made all of them optional cause the advert thats passed down to the props will contain all of these variables from BE but we only display a few of them, and thats the case for some other components where we only use a few of these variables... eslint will complain for me if i don't include them all

Reply:
but ill remove the optional for each of the variables, and just use partial